packit_service/config.py

```diff
@@ -91,8 +92,12 @@ def __init__(
         self.server_name: str = ""
 
         # Makeshift for now to authenticate webhooks coming from gitlab instances
+        # Old way of authenticating
         self.gitlab_webhook_tokens: Set[str] = set(gitlab_webhook_tokens or [])
 
+        # Gitlab token secret to decode JWT tokens
+        self.gitlab_token_secret: str = gitlab_token_secret
```

Comment:
What is the benefit here? Can we avoid having just another value in the config?

Reply:
Is it really used?

Reply:
We are using `gitlab_token_secret` to encode the json payload which in this case its {namespace: "the-namespace", repo_name:"the-repo", project_url: "https://theurl.com"}. eg - https://pyjwt.readthedocs.io/en/latest/#example-usage

With the same secret then we can decode the token and directly check for data from the event (repo_name, Namespace)

Reply:
I can use webhook_secret from the config But wanted to keep things different here.

Reply:
My point is if this info is useful to us and worth specifying another token.

Wouldn't random tokens work as well?

Reply:
> I can use webhook_secret from the config But wanted to keep things different from Github here.

I agree. I would not reuse a different token for another usage.